components/d2l-filter-dropdown/d2l-filter-dropdown-category.js

```diff
@@ -153,7 +157,7 @@ class FilterDropdownCategory extends LocalizeStaticMixin(TabPanelMixin(LitElemen
 				</d2l-input-search>
 			</div>
 			<d2l-menu @d2l-menu-item-change="${this._handleMenuItemChange}" label="${this.text}">
-				<slot></slot>
+				<slot @slotchange="${this._onSlotChange}"></slot>
```

Comment:
I'm assuming you don't need to support IE11 or that this isn't going out in the June release?

Reply:
I don't think we've been supporting ie11 in portfolio, since we're using slotchange elsewhere, maybe I can leave a comment in the readme `applied-filters` section saying that it uses slotchange and won't support ie

Reply:
Yeah a comment would be good, may be unlikely it will start being used elsewhere in the next week and a half but you never know